website/content/project/policies.md

```diff
@@ -0,0 +1,331 @@
+
+Each team will be assigned to work on [an existing or new feature of chiventure]({{< ref "/project/features.md" >}}) (teams will have a chance to express a preference, although we can’t guarantee we’ll be able to assign each team to their preferred feature). In each team, a TA will act as Senior Developer.
+
+
+# Project Timeline
+
+The project will span seven weeks (Weeks 3 - 9), with a final review and team presentations taking place during Finals Week. The seven weeks of the project will be divided into a project kick-off and four sprints, with sprint reviews happening on Mondays.
+
+*   Week 3: Project kick-off. This week will be primarily used for you to familiarize yourself with project and the exact work your team will do.
+*   Week 4: Sprint 1
+*   Week 5: Sprint 2. 
+*   Weeks 6-7: Sprint 3
+*   Weeks 8-9: Sprint 4
+
+
+# Project Evaluation
+
+The project is worth 70% of your final grade, and these 70 points are divided as follows:
+
+*   5: Team Design Exercise
+*   35: Your individual score on the project
+*   15: Your team score (the average of all the individual scores on a team)
+*   5: Peer feedback
+*   10: Final presentation during Finals Week.
+
+Unlike a traditional programming assignment or project, the exact final outcome of the project is not known in advance. Of course, we will set some high-level goals and requirements and, while those requirements will help us measure whether the project is “done”, they are not a strict rubric. As we saw in class, this is one of the strengths (and challenges) of agile software development: you can re-evaluate requirements and make design changes quickly, but there is no detailed requirements document that you can use to check whether you’ve contractually fulfilled your obligations to a customer.
+
+As such, your individual score on the project will be based on how well you perform certain tasks that you will do throughout the project (and which you would encounter in any real software project). More specifically, you will be evaluated based on the following:
+
+*   How well you document tasks/issues 
+*   Whether you complete the tasks you commit to doing in a given sprint
+*   Whether you get pull requests approved
+
+Note: no part of your score will be determined by your teammates' reviews/feedback. So, when you receive a code review or peer feedback, we will be evaluating the quality of the code review / peer feedback itself; we will not be using that review/feedback to determine your score. Please note that, in peer feedback, we will specifically be evaluating the person giving the feedback (by looking at the quality of the feedback they provide).
+
+Each of the above has a point value:
```

Comment:
Could we say "each of these has a different weight"?

Reply:
Fixed